tests/jballerina-unit-test/src/test/resources/test-src/query/query_expr_with_errors.bal

```diff
@@ -485,11 +482,189 @@ function testErrorReturnedFromOrderByClause() {
 }
 
 function checkErrorAtOrderBy() returns error? {
-    _ = from int i in 1...3
+    _ = check from int i in 1...3
        order by check verifyCheck(i)
        select i;
 }
 
```

Comment:
Shall we add query action and expr tests to check https://github.com/ballerina-platform/ballerina-lang/pull/36647/files#r904518822, the group expr with parentheses and double parentheses